File: 01多对象手动内存管理/ViewLearnDemo/ViewController.m
//
//  ViewController.m
//  ViewLearnDemo
//
//  Created by linw on 16/4/20.
//  Copyright © 2016年 linw. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    UIButton *button = [UIButton buttonWithType:UIButtonTypeContactAdd];
    CGRect rect = button.frame;
    rect.origin.y = 200;
    button.frame = rect;
    [self.view addSubview:button];
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift_01_UI_Learn/swift_01_UI_Learn/ViewController.swift
//
//  ViewController.swift
//  swift_01_UI_Learn
//
//  Created by 林威 on 16/5/6.
//  Copyright © 2016年 林威. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var main_image: UIImageView!
    
    
    @IBOutlet weak var nameTextField: UITextField!
    
    
    @IBOutlet weak var passTextField: UITextField!
    
    
    
    @IBAction func loginButton(sender: AnyObject) {
        print(nameTextField.text)
        print(passTextField.text)
        
    }
    override func viewDidLoad() {
        super.viewDidLoad()

        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
        self.reloadInputViews()
        passTextField.resignFirstResponder()
        nameTextField.resignFirstResponder()

    }

}

